src/game/client/videoplayer/videoplayer.cpp

```diff
@@ -0,0 +1,181 @@
+/**
+ * @file
+ *
+ * @author OmniBlade
+ *
+ * @brief Virtual interface for video players.
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "videoplayer.h"
+#include "videostream.h"
+#include <cstddef>
+
+#ifndef THYME_STANDALONE
+VideoPlayer *&g_theVideoPlayer = Make_Global<VideoPlayer *>(0x00A2C0AC);
+#else
+VideoPlayer *g_theVideoPlayer = nullptr;
```

Comment:
Is it being initialised anywhere?

Reply:
Its initialised in GameEngine::Init which is unfinished so the original code currently initialises it.